classes/class-p4-custom-taxonomy.php

```diff
@@ -195,12 +269,30 @@ public function replace_taxonomy_terms_in_rewrite_rules( $rules ) {
 					$term_slugs[] = 'story';
 				}
 
-				$terms_slugs = implode( '|', $term_slugs );
+				return $term_slugs;
+			}
+
+			return false;
```

Comment:
Since this function is defined to return array, I guess we should return [];
Also, please define it as `private function get_terms_slugs() : array {`